### PR TITLE
Евдокимов Артем - Лабораторная работа #2 

### PR DESCRIPTION
**Pull request by black8cherry**

### CODE REVIEW

**Comment by Eskelai** [comment]
:+1:

**Comment by MacLin123** [comment]
👍

File: modules/shapes-volume/test/test_shapes_volume.cpp [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+TEST(Evdokimov_Artem_ShapesVolume , Can_Calculate_Volume_Of_Cylinder) {
+    // Arrange
+    double radius = 2.0;
+    double height = 3.0;
+    // Act
+    double result = ShapesVolume::CylinderVolume(radius , height);
+    // Assert
+    double exp_ans = 37.6991;
+    EXPECT_NEAR(exp_ans , result , 4);
```
Comment (Agricola87) [inline modules/shapes-volume/test/test_shapes_volume.cpp]: Accuracy is too big. Is 0.01 at least

Comment (black8cherry) [reply modules/shapes-volume/test/test_shapes_volume.cpp]: looks like it works

Comment (black8cherry) [reply modules/shapes-volume/test/test_shapes_volume.cpp]: > Accuracy is too rough. Use 0.01 at least

looks like it works

File: modules/shapes-volume/include/shapes_volume.h [hunk 2]
```diff
@@ -0,0 +1,16 @@
+// Copyright 2019 Evdokimov Artem
+
+#ifndef MODULES_SHAPES_VOLUME_INCLUDE_SHAPES_VOLUME_H_
+#define MODULES_SHAPES_VOLUME_INCLUDE_SHAPES_VOLUME_H_
+
+class ShapesVolume {
+ public:
+    static double SphereVolume(double radius);
+    static double CylinderVolume(double radius , double height);
+    static double CubeVolume(double a);
```
Comment (Agricola87) [inline modules/shapes-volume/include/shapes_volume.h]: Please, make variables-arguments in interface more explicit, for example, edgeLength

Comment (black8cherry) [reply modules/shapes-volume/include/shapes_volume.h]: > Please, make variables-arguments in interface more explicit, for example, edgeLength

Done!

**Comment by black8cherry** [comment]
@Agricola87, please, review

**Pull request merged**